src/runtime/auto-prompt-manager-test.js

```diff
@@ -122,9 +122,14 @@ describes.realWin('AutoPromptManager', {}, (env) => {
     miniPromptApiMock.verify();
   });
 
-  function setWinWithoutGtag() {
+  function setWinWithAnalytics(gtag, ga) {
     const winWithNoGtag = Object.assign({}, win);
```

Comment:
Would renaming this const make sense, since it can contain a gtag now?
```suggestion
    const winWithAnalytics = Object.assign({}, win);
```

Reply:
Yes, I agree. Good catch.